mungegithub/mungers/submit-queue_test.go

```diff
@@ -164,13 +165,19 @@ func FailJenkins() jenkins.Job {
 	}
 }
 
+func getJUnit(testsNo int, failuresNo int) []byte {
```

Comment:
Does this belong here or somehow over in the utils? Also does it matter that you are no longer hard coding `time=1234` ?